mh/constants.py

```diff
@@ -148,7 +153,7 @@ def make_binary_server_type_list(is_jap=False):
     return data
 
 
-def make_binary_npc_greeters(is_jap=False):
+def make_binary_npc_greeters(binary_loader, is_jap=False, temporal_slot=current_event_time_slot()):
```

Comment:
`temporal_slot=current_event_time_slot()` will be evaluated as a default value once, is this behaviour intended?

Reply:
It was not intended and has now been fixed, thanks for pointing it out